Parsers/ASimAuditEvent/Parsers/ASimAuditEventVectraCloudAudit.yaml

```diff
@@ -0,0 +1,54 @@
+Parser:
+  Title: Audit Event ASIM parser for VectraCloud Audit Logs Event
+  Version: '0.1'
+  LastUpdated: Mar 17 2023
+Product:
+  Name: Vectra
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing VectraCloud Audit Logs Event in the Audits_Data_CL table to the ASIM Audit Event schema.
+ParserName: ASimAuditEventVectraCloudAudit
+EquivalentBuiltInParser: _ASim_AuditEvent_VectraCloudAudit
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled:bool = false)
+  {
+    Audits_Data_CL
+    | extend
```

Comment:
Just how we added filters in Authentication, we would have to remove that data from this parser. Otherwise same data would get parsed here as well.

Reply:
Addressed.